### ISSUE TITLE
Unable to select texts within Drawer components

### ISSUE THREAD

**Issue by maskin25** [grafana/grafana#44752]
Selecting text within Drawer component is broken. I found this bug is caused by using @react-aria/focus package.  This change was added here https://github.com/grafana/grafana/pull/42571 and this bug is actual for Grafana v8.4.0-49563pre
